omap/modules/base_settings.py

```diff
@@ -68,7 +69,8 @@
         },
     },
 ]
-
+WSGI_APPLICATION = "omap.modules.wsgi.application"
+ASGI_APPLICATION = "sockpuppet.routing.application"
```

Comment:
shouldnt we have our own asgi application here?
we dont use sockpuppet in OMAP, or?

Reply:
@JulianFeinauer Thats a fair point, I just copied those over, I'll fix that.